dashboard/app/mailers/pd/workshop_mailer.rb

```diff
@@ -103,7 +103,7 @@ def exit_survey(workshop, teacher, enrollment)
     mail content_type: content_type,
       from: from_hadi,
       subject: 'How was your Code.org workshop?',
-      to: email_address(@teacher.name, @teacher.email)
+      to: email_address(@enrollment.name || @teacher.name, @teacher.email)
```

Comment:
What if some existing enrollments are like "Mrs. Parker" and then we'll end up sending emails to "Hi Mrs.,"?

I feel like we should create a new first name and last name field and treat the current enrollment name like we'll treat the Display name. Then we use first/last if it exists and if not i.e. for old enrollments use the original name?